### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by Khaled-Alaa-1**
In this project, I made a database based on a diagram:

- Create a file named schema_based_on_diagram.sql
- Add the FK indexes.
- Update readme file

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  id            SERIAL PRIMARY KEY,
+  name          VARCHAR(255),
+  date_of_birth DATE
+);
+
+-- Creating medical_histories table
+CREATE TABLE medical_histories (
+  id         SERIAL PRIMARY KEY,
+  admited_at TIMESTAMP,
+  patient_id INTEGER UNIQUE REFERENCES patients(id),
```
Comment (Davidosky007) [inline schema_based_on_diagram.sql:12]: * As stated in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.) all FOREIGN KEYS should have indexes created. for example  `CREATE INDEX ON medical_histories (patient_id);`

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+CREATE INDEX idx_medical_history_treatments_medical_history_id ON medical_history_treatments (medical_history_id);
+CREATE INDEX idx_medical_history_treatments_treatment_id ON medical_history_treatments (treatment_id);
+
+-- Creating invoices table
+CREATE TABLE invoices (
+  id               SERIAL PRIMARY KEY,
+  total_amount     DECIMAL,
+  generated_at     TIMESTAMP,
+  payed_at         TIMESTAMP,
+  medical_history_id INTEGER UNIQUE REFERENCES medical_histories(id)
```
Comment (Davidosky007) [inline schema_based_on_diagram.sql:39]: * As stated in the [requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/database_from_diagram.md#:~:text=Remember%20to%20add%20the%20FK%20indexes.) all FOREIGN KEYS should have indexes created. Kindly check the example above ⬆️

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
Hi @Khaled-Alaa-1 @rudicarrilloypr 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
* Database created properly 👍 
* Well written PR 👍 
* Good looking readme 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @davidsoky007 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Reem-lab — APPROVED** [review]
# Hi @Khaled-Alaa-1 and @rudicarrilloypr, 👋🏻👋🏻

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights

- [ ]  Followed Gitflow 💯
- [ ]  Descrtipive PR⭐
- [ ] Kudos for working on the functionality🤩
- [ ] No Linters error ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**